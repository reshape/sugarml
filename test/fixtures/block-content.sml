p.
  hello .this is block content.
  wow such. block
  span no html render wow

#someid.wow.
      testing
       testing2

script. try{wow}catch{@2434ew}

p .foo bar.

section
  p.
    wow

script.
  class Wow {
    constructor () {
      super()
    }
  }
