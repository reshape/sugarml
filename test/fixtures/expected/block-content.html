<p>hello .this is block content.
wow such. block
span no html render wow</p><div id="someid" class="wow">testing
 testing2</div><script>try{wow}catch{@2434ew}</script><p>.foo bar.</p><section><p>wow</p></section><script>class Wow {
  constructor () {
    super()
  }
}</script>
